cloud/jenkins/pgo_operator_aws_openshift-4.groovy

```diff
@@ -186,7 +187,7 @@ compute:
   platform:
     aws:
       type: m5.2xlarge
-  replicas: 3
+  replicas: 1
```

Comment:
how do the tests work with 1?

Reply:
we extand it to 3 but after spot instances avaliable.